Src/RE2.Native/RE2Native.cpp

```diff
@@ -146,3 +151,101 @@ extern "C" __declspec(dllexport) int Matches(int regexIndex, String8 text, int f
 	// Return the number of matches found
 	return nextMatchIndex;
 }
+
+// pattern must use RE2 syntax
+extern "C" __declspec(dllexport) void GetRegexSetup(
```

Comment:
>GetRegexSetup [](http://example.com/codeflow?start=38&length=13)

This should probably be integrated into the existing `BuildRegex` function.

Reply:
Moved to backlog: [https://github.com/microsoft/sarif-pattern-matcher/issues/386](https://github.com/microsoft/sarif-pattern-matcher/issues/386)